csharp/old-change-notes/2020-08-26-implicit-array-lengths.md

```diff
@@ -1,5 +1,5 @@
 lgtm,codescanning
-* Inferring the lengths of implicitely sized arrays is fixed. Previously, multi
+* Inferring the lengths of implicitly sized arrays is fixed. Previously, multi
```

Comment:
We probably shouldn't change anything in the old change notes.

Reply:
We have been accepting spelling fixes to change notes in some of the other PRs that were opened, for example, [here](https://github.com/github/codeql/pull/10802/files#diff-784d6a6c025528388cf2190acef01f5076a9ef9e597baf78d9e9d911016c7fb2) and [here](https://github.com/github/codeql/pull/10801/files#diff-218bb605d541548f68e8ef510ee7de4c94215c559d577071fd9fc753c1a25a88), although those changes were not in `old-change-notes` directories. The change note will now of course be slightly out-of-sync with the one in the released version, but not in a way that seems harmful. Unless I completely overlooked something?